src/CodeCoverage.php

```diff
@@ -374,10 +395,11 @@ public function merge(self $that): void
             \array_merge($this->filter->getWhitelistedFiles(), $that->filter()->getWhitelistedFiles())
         );
 
-        foreach ($that->data as $file => $lines) {
+        // I don't know how / why this works, but it should be refactored to ->getData()
+        foreach ($that->getData() as $file => $fileData) {
             if (!isset($this->data[$file])) {
                 if (!$this->filter->isFiltered($file)) {
-                    $this->data[$file] = $lines;
+                    $this->data[$file] = $fileData;
```

Comment:
What are your thoughts about using `getData` on `$this` instead of just `$that`?

And you probably meant this with your above comment, but just to be sure: the below `$that->data` calls should probably be replaced with `$that->getData()`

Reply:
Bingo. Really, I put the comment in before I changed the call in `foreach`. I'd like to follow that implementation - it just feels like the right thing to do.